src/utils/pgwire/src/pg_extended.rs

```diff
@@ -356,105 +218,545 @@ impl PgPortal {
     }
 }
 
+/// default_params is to create default params:[String] from type_description:[TypeOid].
+/// The params produced by this function will be used in the PreparedStatement.
+///
+/// type_description is a list of type oids.
```

Comment:
We should also mention that the default params is used when user do not provide a value?

Reply:
default_params used in instance_default. I add comments in instance_default to illustrate why we need instance_default() and why we use default_params().

Reply:
Good!